moped-editor/src/views/projects/projectsListView/ProjectsListViewFiltersConf.js

```diff
@@ -414,6 +416,18 @@ export const PROJECT_LIST_VIEW_FILTERS_CONFIG = {
         "string_is_not_null",
       ],
     },
+    {
+      name: "project_and_child_project_council_districts",
+      label: "Council Districts",
```

Comment:
This is title case instead of sentence case, but I also noticed that "Has Subprojects" is also title case and I'm pretty sure I did that, somehow thinking subprojects was a proper noun (?!?!).

Reply:
Great catch! I'm about to push up fix to get this right. 🙏